server/src/caching.js

```diff
@@ -7,6 +7,7 @@ if (! config.setCache) {
 }
 
 exports.setCache = function (res, options) {
+  console.log("setcache?", config.setCache, process.env.SET_CACHE);
```

Comment:
This is probably useful to leave in, but it should be an info. Or take it out if you don't think we always need to see this.